packages/cozy-notifications/README.md

```diff
@@ -0,0 +1,190 @@
+`cozy-notifications` provides tools to send notifications (push or email) from a Cozy
+application or konnector.
+
+## `sendNotification`
+
+The main entrypoint of this library is `sendNotifification(cozyClient, notificationView)`.
+
+Before being able to use it, you must define a notification view class.
+
+## Notification views
+
+Notification views are responsible for
+
+- fetching the data necessary for the notification
+- providing a template for the notification
+- configuring the notification (`category`, `channels`)
+- deciding if the notification should be sent
+
+```
+class MyNotification {
+  async buildData() {
+    return {
+      // data that will be used in the template
+    }
+  }
+
+  getPushContent() {
+    return 'push content'
+  }
+
+  getTitle() {
+    return 'title'
+  }
+}
+
+MyNotification.template = require('./template.hbs')
+```
+
+## Templates
+
+Templates used in notification views are based on [`handlebars`](https://handlebarsjs.com/)
+and [`handlebars-layouts`](https://www.npmjs.com/package/handlebars-layouts). This makes
+it   simple to create notifications that are based on built-in templates.
```

Comment:
Should we also talk about MJML? Since we're using a few mjml tags in the exemple?

Reply:
Done